src/avfoundation.cs

```diff
@@ -13024,8 +13513,7 @@ interface AVMutableAssetDownloadStorageManagementPolicy
 
 	[NoWatch]
 	[NoTV]
-	[iOS (9,0)]
-	[NoMac]
+	[iOS (9,0), Mac (12,0)]
```

Comment:
catalyst ?

Reply:
same, it is the min version of catalyst.